vllm/spec_decode/spec_decode_worker.py

```diff
@@ -725,6 +736,15 @@ def _create_output_sampler_list(
         if maybe_rejsample_metrics is not None:
             sampler_output_list[
                 0].spec_decode_worker_metrics = maybe_rejsample_metrics
+
+            (average_time_per_proposal_tok_ms, scoring_time_ms,
+             verification_time_ms) = stage_times
+            logger.info(
+                "SpecDecodeWorker stage times: "
+                "average_time_per_proposal_tok_ms=%.02f "
+                "scoring_time_ms=%.02f verification_time_ms=%.02f",
+                average_time_per_proposal_tok_ms, scoring_time_ms,
+                verification_time_ms)
```

Comment:
It looks like we will print this log every step? Did I miss something?

Reply:
only when rejection sampler emits metrics, which is every 5s

Reply:
I'll add comment